src/main/java/Output.java

```diff
@@ -0,0 +1,30 @@
+public class Output {
+
+    //Coding standard: start every method with "print" for readability
+
+    public static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printLogo() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println(logo);
+    }
+    public static void printGreet() {
```

Comment:
logo and greeting could be included in one method, make code cleaner!

Reply:
Wanted to abstract a logo method out in case I have to print the Logo elsewhere later on in the app. Thanks for the suggestion. Not sure if I will need it though, thanks for the suggestion!